lint.js

```diff
@@ -97,7 +97,7 @@ const command = async (file, cmd) => {
         verbose
     });
 
-    validator.validate(spec, { verbose, skip: cmd.skip, lint: true, linter: linter.lint }, (err, _options) => {
+    validator.validate(spec, { verbose, skip: cmd.skip, lint: true, linter: linter.lint, prettify: true }, (err, _options) => {
```

Comment:
Did you consider setting `options.validateSchema` to `'first'`? That will give you `better-ajv-errors` in preference to the hand-rolled `should.js` assertions from `oas-validator` in cases where the validation checks overlap.

It looks like you might have done for the screenshot you posted on Twitter? Unless the custom validator missed the added `nonsense` property...

That said, I find they sometimes produce `misleading-ajv-errors` (like `$ref should be here` when it's actually failing a `oneOf` constraint and not telling you about the other branch), and there seems to be at least one bug in their JSON-Path handling: https://github.com/atlassian/better-ajv-errors/issues/23